lib/ParamedicKill.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var shelljs = require("shelljs");
+var util    = require("./utils").utilities;
+var logger  = require('./utils').logger;
+
+function ParamedicKill(platform) {
+    this.platform = platform;
+}
+
+ParamedicKill.prototype.kill = function() {
+    // shell config
+    shelljs.config.fatal  = false;
+    shelljs.config.silent = false;
+
+    // get platform tasks
+    var platformTasks = this.tasksOnPlatform(this.platform);
+
+    if (platformTasks.length < 1) {
+        console.warn("no known tasks to kill");
+    }
+
+    // kill them
+    this.killTasks(platformTasks);
+
+    if (this.platform === util.ANDROID) {
+        this.killAdbServer();
+    }
+
+}
+
+ParamedicKill.prototype.tasksOnPlatform = function (platformName) {
+    switch (platformName) {
+    case util.WINDOWS:
+        return ["WWAHost.exe", "Xde.exe"];
+    case util.IOS:
+        return ["Simulator"];
+    case util.ANDROID:
+        if (util.isWindows()) {
+            return ["emulator-arm.exe"];
+        } else {
+            return ["emulator64-x86", "emulator64-arm"];
+        }
+        break;
```

Comment:
Personally I prefer NOT returning from a switch, and having a break for every condition ( or a commented fall through )

```
var retTaskList = [];
switch(platformName) {
  case util.WINDOWS :
      retTaskList = ["WWAHost.exe", "Xde.exe"];
      break;
 ...
} 
return retTaskList;
```